src/engraving/rendering/engravingitempreviewpainter.cpp

```diff
@@ -23,14 +23,21 @@
 #include "engravingitempreviewpainter.h"
 
 #include "engraving/dom/actionicon.h"
+#include "engraving/dom/chord.h"
+#include "engraving/dom/note.h"
+#include "engraving/dom/stem.h"
+
+#include "engraving/dom/factory.h"
+
 #include "engraving/style/defaultstyle.h"
 #include "engraving/dom/masterscore.h"
 
-using namespace mu::engraving::rendering;
+using namespace mu::engraving;
+using namespace rendering;
```

Comment:
We tend to write subnamespaces out in full, and I think that's good for clarity:
```suggestion
using namespace mu::engraving;
using namespace mu::engraving::rendering;
```